# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/rental/Cargo.toml
# Commit: 16350e08d34f
# Full Hash: 16350e08d34ff06c7f46745c6b160c7a7bda9140
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2019-12-06 04:43:57
# Regressor Bug: 1586855
# File Overlap Count: 6
# Description:
#   bug 1586855 - incorporate CRLite filters into cert_storage r=jcj,kjacobs
#   
#   This patch implements CRLite lookups for TLS server certificate revocation
#   information in telemetry-only mode. It adds a new preference
#   "security.pki.crlite_mode" to control the behavior of this feature. Setting
# ==============================================================================

diff -r 64fd40663930 -r 16350e08d34f third_party/rust/rental/Cargo.toml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/third_party/rust/rental/Cargo.toml	Thu Dec 05 22:41:53 2019 +0000
@@ -0,0 +1,33 @@
+# THIS FILE IS AUTOMATICALLY GENERATED BY CARGO
+#
+# When uploading crates to the registry Cargo will automatically
+# "normalize" Cargo.toml files for maximal compatibility
+# with all versions of Cargo and also rewrite `path` dependencies
+# to registry (e.g., crates.io) dependencies
+#
+# If you believe there's an error in this file please file an
+# issue against the rust-lang/cargo repository. If you're
+# editing this file be aware that the upstream Cargo.toml
+# will likely look very different (and much more reasonable)
+
+[package]
+name = "rental"
+version = "0.5.5"
+authors = ["Jameson Ernst <jameson@jpernst.com>"]
+description = "A macro to generate safe self-referential structs, plus premade types for common use cases."
+documentation = "https://docs.rs/rental"
+readme = "README.md"
+keywords = ["lifetime", "ownership", "borrowing", "self", "reference"]
+categories = ["rust-patterns", "no-std"]
+license = "MIT/Apache-2.0"
+repository = "https://github.com/jpernst/rental"
+[dependencies.rental-impl]
+version = "=0.5.5"
+
+[dependencies.stable_deref_trait]
+version = "1.0.0"
+default-features = false
+
+[features]
+default = ["std"]
+std = ["stable_deref_trait/std"]